packages/sprite/src/Sprite.ts

```diff
@@ -36,10 +38,37 @@ const indices = new Uint16Array([0, 1, 2, 0, 2, 3]);
  */
 export class Sprite extends Container
 {
+    public blendMode: BLEND_MODES;
+    public shader: Filter|Shader;
```

Comment:
Sprites dont have usable `shader` field at the moment. Its old thing from v3. I do not know how did it survive.

Reply:
cool, I've removed shader